lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -245,7 +245,7 @@ class TestFieldOverrideMongoPerformance(FieldOverridePerformanceTestCase):
     __test__ = True
 
     # TODO: decrease query count as part of REVO-28
-    QUERY_COUNT = 31
+    QUERY_COUNT = 33
```

Comment:
Query count has increased by 2 since we now have a query to check if discussions are enabled twice, once by the standard discussion tab, and the other time by the LTI-based tab. This can potentially be cached by DiscussionConfig.get?